src/bin/scribble.ts

```diff
@@ -550,8 +570,67 @@ if ("version" in options) {
     const groupsMap: Map<string, SourceUnit[]> = new Map();
     const ctxtsMap: Map<string, ASTContext> = new Map();
     const filesMap: Map<string, Map<string, string>> = new Map();
-    const originalFiles: Set<string> = new Set();
-    const instrumentationFiles: Set<string> = new Set();
+
+    let metaDataFile: string;
+    let originalFiles: Set<string>;
+    let instrumentationFiles: Set<string>;
+
+    if (options["instrumentation-metadata-file"]) {
```

Comment:
Maybe move the logic for detecting the location of `instrumentation-metadata-file` in own helper function? Something like:

```
function detectInstrMetdataFilePath(options: any): string {
     ...
}```

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/133/commits/62dfc4eda8b6d526ae493944f6749a62b738b079